burn-book/src/building-blocks/tensor.md

```diff
@@ -186,6 +186,7 @@ Those operations are available for numeric tensor kinds: `Float` and `Int`.
 | `tensor.argmin(dim)`                                            | `tensor.argmin(dim)`                           |
 | `tensor.argsort(dim)`                                           | `tensor.argsort(dim)`                          |
 | `tensor.argsort_descending(dim)`                                | `tensor.argsort(dim, descending=True)`         |
+| `tensor.argtopk(k, dim)`                                        | `tensor.topk(k, dim).indices`                  |
```

Comment:
We have `tensor.topk_with_indices(k, dim)` where you can easily ignore the returned values and only keep the indices. I would not add `tensor.argtopk` as an operation.

Reply:
Yeah that's fair enough, I just added it because I had a use for just the indices and since sort, sort_with_indices and argsort, exited I thought of doing the same for topk. Does it not make a difference (in terms of resources) to just produce the top k values versus the values and it's indexes?

Reply:
I think it is not as common and easily achieved with a combination of two ops (as you implemented). That's why I don't think adding this to the tensor operations is not necessary, otherwise the number of operations could become very crowded.

Regarding resources, the difference is probably minimal, but that depends on the size of the tensor. So you can either use `top_k_with_indices` while discarding the values or `argsort_descending` + `select` as in your implementation.

Reply:
alrighty, removed